src/validation.cpp

```diff
@@ -2248,6 +2250,22 @@ bool BitcoindRPCCheck(const bool init)
                     }
                 }
                 UniValue result = reply["result"];
+                if (!result.isObject() || !result.get_obj()["version"].isNum() ||
+                        result.get_obj()["version"].get_int() < MIN_PARENT_NODE_VERSION) {
+                    LogPrintf("ERROR: Parent chain daemon too old; "
+                            "need BitcoinCore version 0.16.2 or newer.\n");
```

Comment:
Space between Bitcoin and Core, sorry :)

Reply:
Whoops :/